cmd/main.go

```diff
@@ -33,6 +34,7 @@ func main() {
 
 	// TODO: replace cloud -> evpn
 	pb.RegisterCloudInfraServiceServer(s, opi)
```

Comment:
I guess also this one needs to be removed

Reply:
correct, once we complete transition to new APIs, old will be removed